src/main/java/net/pms/util/OpenSubtitle.java

```diff
@@ -214,14 +218,21 @@ public static String computeHash(Path file) throws IOException {
 			return null;
 		}
 
+		String cachedHash = cacheUtil.getItem(file.getFileName().toString());
+		if(cachedHash != null){
```

Comment:
The last comment. This should be if (cachedHash != null) {``

Reply:
Let me fix that right away. Thanks for spotting. Also, is there a linter configuration specific to this project that I can add to my IntelliJ to avoid such formatting errors?

Reply:
We have that for Eclipse in the root folder of this project.

Reply:
I will take a look and see if I can port that to IntelliJ. Thanks for the info. The last format request is now complete.